testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -76,12 +77,19 @@
     private String keyspace;
     private static CqlSession session;
 
+    private static final int KEYSPACE_NAME_MAX_LENGTH = 48;
+
     @Before
     public void setup()
     {
         String testName = name.getMethodName();
         testName = testName.substring(0, testName.indexOf("["));
         keyspace = "ks_" + new Date().getTime() + "_" + testName;
+
+        if (keyspace.length() > KEYSPACE_NAME_MAX_LENGTH) {
```

Comment:
It may be dangerous - if there will be 2 long tests with the same suffix, they will end up in the same keyspace. Maybe we can randomize the last 2 characters or something similar?

Reply:
It uses the timestamp as prefix. The table will still be different, I'm just limiting when reaching maximum.